server/src/internalClusterTest/java/org/elasticsearch/indices/DataStreamIT.java

```diff
@@ -115,23 +116,23 @@ public void testBasicScenario() throws Exception {
         assertThat(getDataStreamResponse.getDataStreams().get(0).getTimeStampField().getFieldMapping(), equalTo(Map.of("type", "date")));
         assertThat(getDataStreamResponse.getDataStreams().get(0).getIndices().size(), equalTo(1));
         assertThat(getDataStreamResponse.getDataStreams().get(0).getIndices().get(0).getName(),
-            equalTo(DataStream.getDefaultBackingIndexName("metrics-bar", 1)));
```

Comment:
This caused a lot of unrelated changes. Maybe not statically import `getDefaultBackingIndexName()` method?